packages/storybook/stories/QuickStarts/snippets/QuickstartCompositeAdapter.snippet.tsx

```diff
@@ -8,19 +8,32 @@ import {
 } from '@azure/communication-react';
 import React, { useEffect, useMemo, useState } from 'react';
 
+export type CompositeProps = {
+  userId: string; // '<Azure Communication Services Identifier>'
+  token: string; // '<Azure Communication Services Access Token>'
+  endpoint: string; // '<Azure Communication Services Resource Endpoint>'
+  displayName: string; // '<Display Name>'
+  groupId: string; // '<Developer generated GUID>'
+  threadId: string; // '<Get thread id from chat service>'
+};
+
+const compositeProps: CompositeProps = {
+  displayName: '<Display Name>',
+  userId: '<Azure Communication Services Identifier>',
+  token: '<Azure Communication Services Access Token>',
+  endpoint: '<Azure Communication Services Resource Endpoint>',
+  threadId: '<Get thread id from chat service>',
+  groupId: '<Developer generated GUID>'
+};
```

Comment:
I got the concern of making all options/params for composite more obvious into one place, which is great and makes a lot of sense
For creating an object for all these variables, I would say this one is more like an *App* instead of a *component*, so define a type and create a single object to contain everything, I would say either make it a real component like this:

```
type AppProps = {

....

};

function App(props: AppProps): JSX.Element {


}
ReactDOM.render(<App [pass all those props here] />, domElement);
```
or have a comment block to put all parameters at the top(I prefer this one, because it is neat), creating an object is the best practice for App and might confuse our code reader (especially when they come for quick start)

Reply:
Would like to see some opinions from others, and also listen to any other concerns to wrap parameters into an object instance

Reply:
And one question to be discussed as well, since this is the first example developers will see, we will never be too careful. Do we want to expose all parameter in the very first example? It makes our code super long and might scare some potential customers away - we could show a super short version first and have a long version hiding somewhere if they wanna do more exploration

Reply:
I could definitely remove all the optional args from the object if we wanted to leave those for the specific examples in the respective composite pages. Having it as a actual prop to the App.tsx file though does bring it back around to the issue of making them edit the index.tsx file. I'm not too sure myself if thats something we want them to do if we are aiming to make this as simple as possible. Maybe a comment or something about the type saying that its a example of a props type if the composites were to be children instead of on the app element?